its/src/test/java/com/sonar/it/scanner/msbuild/sonarcloud/IncrementalPRAnalysisSonarCloudTest.java

```diff
@@ -0,0 +1,221 @@
+/*
+ * SonarScanner for .NET
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package com.sonar.it.scanner.msbuild.sonarcloud;
+
+import com.sonar.it.scanner.msbuild.utils.TestUtils;
+import com.sonar.it.scanner.msbuild.utils.VstsUtils;
+import com.sonar.orchestrator.http.HttpException;
+import com.sonar.orchestrator.util.Command;
+import com.sonar.orchestrator.util.CommandExecutor;
+import com.sonar.orchestrator.util.StreamConsumer;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+public class IncrementalPRAnalysisSonarCloudTest {
+  private final static Logger LOG = LoggerFactory.getLogger(IncrementalPRAnalysisSonarCloudTest.class);
+  private final static Integer COMMAND_TIMEOUT = 2 * 60 * 1000;
+  private final static String SCANNER_PATH = System.getenv("SCANNER_PATH") == null
+    ? "../build/sonarscanner-msbuild-net46/SonarScanner.MSBuild.exe" // On the local machine, the scanner is prepared by ci-build.ps1 script.
+    : System.getenv("SCANNER_PATH");
+  private final static String[] prArguments = {
+    "/d:sonar.pullrequest.base=master",
+    "/d:sonar.pullrequest.branch=pull-request-branch",
+    "/d:sonar.pullrequest.key=pull-request-key"
+  };
+  private final static String SONARCLOUD_ORGANIZATION = System.getenv("SONARCLOUD_ORGANIZATION");
```

Comment:
Educational:
Locally, we have to set all these env vars ourselves if I am not mistaken.
How does the VM have these settings set?

Reply:
Yes, we have to manually configure them locally, either by setting them on the operating system or by configuring them in the IDE. I personally prefer the IDE.

Regarding the VM, I've configured them on Azure DevOps and hey are loaded using https://github.com/SonarSource/sonar-scanner-msbuild/blob/master/azure-pipelines.yml#LL21C12-L21C42

Reply:
I would suggest document it in this PR, to not forget about it. 
Especially since it's a "breaking" change and you need the environment variables to run the ITs.